src/components/Textarea/index.tsx

```diff
@@ -3,9 +3,24 @@ import styled from '@emotion/styled'
 import { useRef } from 'react'
 
 export interface TextAreaProps extends HTMLAttributes<HTMLTextAreaElement> {
+  /** TextArea 컴포넌트 상단의 label 정합니다.
+   * @type string
+   */
   label?: string
+
+  /** TextArea 컴포넌트 하단의 guideMessage 정합니다.
+   * @type string
+   */
   guideMessage?: string
+
+  /** TextArea 컴포넌트의 type을 정합니다.
+   * @type 'filled' | 'ghost'
+   */
```

Comment:
```suggestion
  /** TextArea 컴포넌트의 보여질 형태를 정합니다.
   * @type 'filled' | 'ghost'
   */
```
요기도 한글로 의미 전달하면 좋을 것 같아용!